sdk/core/azure-core/inc/azure/core/core.hpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+/**
+ * @file core.hpp
```

Comment:
nit: remove file name from comment.
```suggestion
 * @file
```